doc/jupyterlab/sharing.md

```diff
@@ -0,0 +1,36 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Share notebooks with the click of a button
+    keywords: jupyter, jupyterlab
+    property=og:locale: en_US
+---
+
+# Sharing notebooks
+
+```{versionadded} 0.1.6
+Notebook sharing requires `jupysql-plugin>=0.1.6`
+```
+
+You can share notebooks with the click of a button using our JupyterLab integration.
```

Comment:
We can add here why and when sharing notebooks will be useful

Reply:
good call, added